src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -127,27 +137,65 @@ public Feedback detailText(@Nullable String detailText) {
     }
 
     /**
-     * sets the detail text of the feedback. In case the detail text is longer than 5000 characters, the additional characters are cut off to avoid database issues
+     * Sets the detail text of the feedback.
```

Comment:
I would mention the rough logic of the method here, in particular if feedback is larger than FEEDBACK_DETAIL_TEXT_MAX_CHARACTERS, only FEEDBACK_DETAIL_TEXT_MAX_CHARACTERS will be stored in the detailText and reference to a new object LongFeedbackText with the full / remaining characters will be created